src/routes/lobby/components/LobbyPlayerIndicator.vue

```diff
@@ -1,22 +1,21 @@
 <template>
   <div v-if="playerUsername" class="player-card">
     <span class="player-name">{{ playerUsername }}</span>
-    <div class="card-container">
+    <div class="card-container" :class="[playerReady ? 'ready' : 'notReady']">
       <img
         src="/img/cards/card-ready.png"
         class="card-front"
-        :class="{ 'ready': playerReady }"
         alt="card front"
         data-cy="lobby-ready-card"
       >
       <img
         src="/img/cards/card-back.png"
         class="card-back"
-        :class="{ 'ready': playerReady }"
         alt="card back"
         data-cy="lobby-back-card"
       >
     </div>
+    <p>{{ playerReady }}</p>
```

Comment:
Just making a note that this is only for development/debugging purposes. Totally reasonable. We can also use the vue devtools to inspect the variables